### PR TITLE
fix: `onlyAvailable` property wrong type

### PR DESCRIPTION
**Pull request by KevLehman**

### CODE REVIEW

File: apps/meteor/tests/data/users.helper.js [hunk 1]
```diff
@@ -86,3 +83,9 @@ export const setUserActiveStatus = (userId, activeStatus = true) =>
 			})
 			.end(resolve);
 	});
+
+export const setUserStatus = (overrideCredentials = credentials, status = 'online') =>
```
Comment (murtaza98) [inline apps/meteor/tests/data/users.helper.js:87]:
```suggestion
export const setUserStatus = (overrideCredentials = credentials, status = UserStatus.ONLINE) =>
```

File: apps/meteor/tests/end-to-end/api/livechat/01-agents.ts [hunk 2]
```diff
@@ -114,6 +114,64 @@ describe('LIVECHAT - Agents', function () {
...
+				.expect('Content-Type', 'application/json')
+				.query({ showIdleAgents: true })
+				.expect(200)
+				.expect((res: Response) => {
+					expect(res.body).to.have.property('success', true);
+					expect(res.body.users).to.be.an('array');
+					expect(res.body).to.have.property('offset');
+					expect(res.body).to.have.property('total');
+					expect(res.body).to.have.property('count');
+					expect(res.body.users.every((u: { status: string }) => !u.status || ['online', 'offline', 'away', 'busy'].includes(u.status))).to
```
Comment (murtaza98) [inline apps/meteor/tests/end-to-end/api/livechat/01-agents.ts:152]:
```suggestion
					expect(res.body.users.every((u: { status: string }) => !u.status || [UserStatus.ONLINE, UserStatus.OFFLINE, UserStatus.AWAY, UserStatus.BUSY].includes(u.status))).to
```

File: apps/meteor/tests/end-to-end/api/livechat/01-agents.ts [hunk 3]
```diff
@@ -114,6 +114,64 @@ describe('LIVECHAT - Agents', function () {
...
+						.be.true;
+				});
+		});
+
+		it('should return only online agents when showIdleAgents is false', async () => {
+			await setUserStatus(agent2.credentials, 'online');
```
Comment (murtaza98) [inline apps/meteor/tests/end-to-end/api/livechat/01-agents.ts:158]:
```suggestion
			await setUserStatus(agent2.credentials, UserStatus.ONLINE);
```

File: apps/meteor/tests/end-to-end/api/livechat/01-agents.ts [hunk 4]
```diff
@@ -114,6 +114,64 @@ describe('LIVECHAT - Agents', function () {
...
+				.expect('Content-Type', 'application/json')
+				.query({ showIdleAgents: false })
+				.expect(200)
+				.expect((res: Response) => {
+					expect(res.body).to.have.property('success', true);
+					expect(res.body.users).to.be.an('array');
+					expect(res.body).to.have.property('offset');
+					expect(res.body).to.have.property('total');
+					expect(res.body).to.have.property('count');
+					expect(res.body.users.every((u: { status: string }) => u.status !== 'offline')).to.be.true;
```
Comment (murtaza98) [inline apps/meteor/tests/end-to-end/api/livechat/01-agents.ts:171]:
```suggestion
					expect(res.body.users.every((u: { status: string }) => u.status !== UserStatus.OFFLINE)).to.be.true;
```

File: apps/meteor/tests/data/users.helper.js (hunk 1 above)
Comment (KevLehman) [reply apps/meteor/tests/data/users.helper.js:87]: That file doesn't look related to these types of changes

Comment (murtaza98) [reply apps/meteor/tests/data/users.helper.js]: I vaguely remember that this was created also as a helper for users related actions like login...so setting a user's status should fall into this area

**Review by aleksandernsilva — APPROVED** [review]

**Review by debdutdeb — APPROVED** [review]

**Review by pierre-lehnen-rc — APPROVED** [review]

**Review by tassoevan — APPROVED** [review]

**Pull request merged**